modules/migration-understanding-resources-mtc-migrates.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * migration/migrating_3_4/migrating-application-workloads-3-4.adoc
+// * migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
+
+[id='about-resources-mtc-migrates_{context}']
+= About the resources migrated
+[role="_abstract"]
+
+You can use the {mtc-full} ({mtc-short}) to migrate the following types of resources:
```

Comment:
No, we are not telling the user what they can do. We are telling them the limitations of MTC. This sentence should be declarative: MTC migrates the following resources.